packages/core/package.json

```diff
@@ -96,8 +96,8 @@
     "dependencies": {
         "@babel/polyfill": "^7.12.1 ",
         "@datadog/browser-rum": "^5.11.0",
-        "@deriv-com/analytics": "1.28.2",
-        "@deriv-com/auth-client": "^1.3.10",
+        "@deriv-com/analytics": "1.30.2",
```

Comment:
The RC branch could be outdated, because in master it matches:
<img width="470" alt="Screenshot 2025-02-07 at 10 29 20 PM" src="https://github.com/user-attachments/assets/cd3adf2d-2ca2-4853-bcaa-e7d5b8087a89" />

Reply:
Got it, will omit out this change